live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/metadata/IngestMetadataCommandHandlerTest.java

```diff
@@ -132,4 +132,46 @@ void shouldSetNewlyCreatedNodeMetadataProperties()
         assertTrue(updateNodeMetadataEvent.getMetadataPropertiesToUnset().isEmpty(), "There should be no properties to unset");
     }
 
+    @Test
+    void canSupportEventsWithNullUsers()
+    {
+        // given
+        IngestMetadataCommand command = new IngestMetadataCommand(
+                EVENT_TIMESTAMP,
+                NODE_ID,
+                updated(NODE_NAME),
+                updated(NODE_PRIMARY_ASSOC_Q_NAME),
+                updated(NODE_TYPE),
+                updated(null),
```

Comment:
It's hard to tell what this `null` value stands for - maybe introducing some constant like `CREATED_BY_NULL` would be better 🤔

Reply:
Or introducing builder pattern in `IngestMetadaCommand` could be even better 🤔

Reply:
I've just updated the test to make it a little clearer. I don't have an objection if you want to work on a builder pattern replacement though.